### ISSUE TITLE
Erroring when trying to download.

### ISSUE THREAD

**Issue by frenchiveruti** [Vazkii/CMPDL#14]
~ Starting magical modpack download sequence ~
URLs: http://mods.curse.com/modpacks/minecraft/254433-invasion/files/latest http://mods.curse.com/modpacks/minecraft/254433-invasion/files/latest
Modpack filename is latest
CMPDL Temp Dir is /home/claudia/.cmpdl_temp
Modpack temporary directory is /home/claudia/.cmpdl_temp/latest
Downloading zip file latest.zip
Error: Server returned HTTP response code: 403 for URL: http://mods.curse.com/modpacks/minecraft/254433-invasion/files/latest
sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1876)
sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1474)
java.net.URL.openStream(URL.java:1045)
vazkii.cmpdl.CMPDL.downloadFileFromURL(CMPDL.java:296)
vazkii.cmpdl.CMPDL.setupModpackMetadata(CMPDL.java:119)
vazkii.cmpdl.CMPDL.downloadFromURL(CMPDL.java:65)
vazkii.cmpdl.OperatorThread.run(OperatorThread.java:17)
~ Starting magical modpack download sequence ~
URLs: https://addons-origin.cursecdn.com/files/2402/833/Invasion-1.0.4.zip/files/latest https://addons-origin.cursecdn.com/files/2402/833/Invasion-1.0.4.zip/files/latest
Modpack filename is latest
CMPDL Temp Dir is /home/claudia/.cmpdl_temp
Modpack temporary directory is /home/claudia/.cmpdl_temp/latest
Downloading zip file latest.zip
Error: https://addons-origin.cursecdn.com/files/2402/833/Invasion-1.0.4.zip/files/latest
sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1872)
sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1474)
sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:254)
java.net.URL.openStream(URL.java:1045)
vazkii.cmpdl.CMPDL.downloadFileFromURL(CMPDL.java:296)
vazkii.cmpdl.CMPDL.setupModpackMetadata(CMPDL.java:119)
vazkii.cmpdl.CMPDL.downloadFromURL(CMPDL.java:65)
vazkii.cmpdl.OperatorThread.run(OperatorThread.java:17)

**Comment by frenchiveruti** [comment]
Ok solved it, used "project" URL instead of mods curseforge one.
https://minecraft.curseforge.com/projects/invasion

**Comment by thePalindrome** [comment]
We may want to consider silently replacing mods.curse.com with minecraft.curseforge.com and using the numeric ID to make sure nothing weird happens. I've accidently fed it the curse url far too many times :P

**Comment by frenchiveruti** [comment]
Yeah or a notice that just pops up when the word "/projects" is not found.
